packages/astro/src/integration/create-integration.ts

```diff
@@ -144,6 +144,12 @@ function createIntegration<Params extends HotloadAstroClerkIntegrationParams>()
             }`,
           );
         },
+        'astro:config:done': ({ injectTypes }) => {
+          injectTypes({
+            filename: 'types.d.ts',
+            content: `/// <reference types="@clerk/astro/env" />`,
+          });
```

Comment:
This feature is available in Astro >=4.15.0 and will reduce our Astro quickstart by [1 step](https://clerk.com/docs/quickstarts/astro#add-type-script-declarations).

Reply:
so cool 😎